app/javascript/src/js/admin/queue_status_submit.js

```diff
@@ -49,9 +49,11 @@ domready(function() {
     //debugger;
     // And warn the user, with info for developer in console
     var detail = event.detail;
-    var data = detail[0], status = detail[1], xhr = detail[2];
-    console.error("DigitizationQueueItem AJAX status change failed\n\n" + status + " " + data);
-    alert("Uh oh, digitization Queue Item status change failed!");
+    var data = JSON.stringify(detail[0], null, 2);
```

Comment:
It's coming accross as a String that needs to be parsed into json instead of a JS object already? i'm surprised, but I believe you!

Reply:
Oh no, that's not me parsing it. I just wanted to pretty-print the JS object.

Reply:
https://codingwithspike.wordpress.com/2016/11/08/pretty-printing-javascript-objects/

Reply:
Oh i see, it's not parsing, it's actually the reverse, serializing to string. ok!